ecosystem/sep-0010.md

```diff
@@ -121,6 +125,14 @@ Every other HTTP status code will be considered an error. For example:
 }
 ```
 
+### Pooled Accounts
+
+In the context of Stellar, pooled accounts are Stellar accounts that are held and managed by an entity that is _not_ the owner of the funds within the account. An alternative approach to pooled accounts that still allows the client entity to manage and hold account secret keys is to create a separate Stellar account for each unique user. While this is possible, a much more common practice is to hold the funds of many users in a single _pooled_ account.
+
+The `memo` parameter supported in `GET <WEB_AUTH_ENDPOINT>` API calls can be used to create `account`-`memo` composite identifiers for the web service's users. The decoded contents of the token returned from `POST <WEB_AUTH_ENDPOINT>` calls should contain both the `account` and `memo` values so the service requiring SEP-10 authentication can tie each request to the user.
+
+In the future, there may be a more streamlined way to specify users within pooled accounts. This section and the API specification will be updated when that change happens.
```

Comment:
I think this section needs to be used to the bottom of the document because it is informational discussion. Right now it is sitting in between the challenge and token sections which should be back to back.

I also think we should remove the discussion about fund ownership, secret keys, unique user's, the new concept of "pooled accounts", and the theories about what might happen in the future. I think we should call out in this section why memos are relevant and when they can be used. This aligns more specifically with the raw constructs.

```suggestion
### Memos

Stellar transactions may be specified with a memo and services that share a single account for many users typically use the memo as a method to distinguishing one user from another, or one internal off-network account from another.

The `memo_type` and `memo` parameters supported in `GET <WEB_AUTH_ENDPOINT>` API calls can be used to communicate to the server that the client intends to authenticate a Stellar account with an additional claim that the authenticating user is the provided `memo`.
```

Reply:
I think its important to have a section header that hints at what it enables. We don't need to strictly defined Pooled Accounts, but a person looking for how a pooled account is authenticated will appreciate the more direct label.

Reply:
Multi-user accounts is ambiguous with multi-signer, which is a different. I think some of the issues with names like multi-user, pooled, custodial, shared, etc is that they are not terms used by anyone else. In our other SEPs that ellude to the same use cases we discuss only memos, e.g. SEP-2. For this reason I think titling this as memos is sufficient, especially with the link from the summary.

Reply:
Also if muxed addresses take off ultimately it will become the term people use and this current functionality will fade into the background.